src/test/scala/centaur/StandardTestCaseSpec.scala

```diff
@@ -28,8 +28,24 @@ class StandardTestCaseSpec extends FlatSpec with Matchers with ParallelTestExecu
   }
 
   def executeStandardTest(testCase: StandardTestCase, f: Workflow => Test[_]): Unit = {
-    it should s"${testCase.testFormat.testSpecString} ${testCase.workflow.name}" in {
-      f(testCase.workflow).run.get
+
+    // Make tags. Enforce lowercase:
+    // TODO: After #45 merges, can replace the testFormat.toString.stripSuffix("Test") with just .name -
+    val tags = (testCase.tagStrings :+ testCase.workflow.name :+ testCase.testFormat.toString.stripSuffix("Test")) map { x => Tags.getTag(x.toLowerCase) }
+
+    tags match {
+      case Nil =>
+        it should s"${testCase.testFormat.testSpecString} ${testCase.workflow.name}" in {
+          f(testCase.workflow).run.get
```

Comment:
I think you can abstract these `f(testCase.workflow).run.get` calls out and then just call the tests like `it should dosomestuff in myAbstractedFunctionValName`